zerver/webhooks/airbyte/view.py

```diff
@@ -0,0 +1,80 @@
+# Webhooks for external integrations.
+
+from django.http import HttpRequest, HttpResponse
+
+from zerver.decorator import webhook_view
+from zerver.lib.response import json_success
+from zerver.lib.typed_endpoint import JsonBodyPayload, typed_endpoint
+from zerver.lib.validator import WildValue, check_bool, check_int, check_string
+from zerver.lib.webhooks.common import check_send_webhook_message
+from zerver.models import UserProfile
+
+AIRBYTE_TOPIC_TEMPLATE = "{workspace} - {connection} - {source} - {destination}"
+AIRBYTE_MESSAGE_TEMPLATE = (
+    "**Job {jobId} {status}** in {durationFormatted}.\n\n"
+    "**Details:**\n"
+    "* **Source:** {source}\n"
+    "* **Destination:** {destination}\n"
+    "* **Bytes Emitted:** {bytesEmittedFormatted}\n"
+    "* **Bytes Committed:** {bytesCommittedFormatted}\n"
+    "* **Records Emitted:** {recordsEmitted}\n"
+    "* **Records Committed:** {recordsCommitted}"
+)
+AIRBYTE_ERROR_MESSAGE_TEMPLATE = "**Job {jobId} failed.**\n" "* **Error Message:** {errorMessage}"
```

Comment:
I don't think it's right to skip all of the details and only show the error message here, especially since there are no links to follow.

Reply:
I get your message @pranasziaukas. So, in the case of an error during sync, the details of the job should still be displayed but with an error message attached. Right?

Reply:
Exactly, @aniebietafia 💯

Reply:
@pranasziaukas I have incorporated the changes. Do mind reviewing this PR whenever you have the time? Thanks